### ISSUE TITLE
[Bug] plq_Ridge_Regressor raises Exception for 'mse' loss

### ISSUE THREAD

**Issue by BenViolet6** [softmin/ReHLine-python#37]
## Bug Description
I am trying to use plq_Ridge_Regressor with standard Mean Squared Error ('mse'). However, the code raises an Exception in _make_loss_rehline_param, stating that the loss function is not supported.

It seems the string matching for the loss name 'mse' is missing or incorrect in the Python wrapper logic, preventing usage of this basic regression loss.

## Expected Behavior
The model should successfully initialize and fit using the 'mse' (Mean Squared Error) loss function. This is the most fundamental regression loss and should be supported.

## Actual Behavior
The code raises an Exception during clf.fit(X=X, y=y). The traceback shows the error occurs inside _make_loss_rehline_param, with the message: Exception: Sorry, ReHLine currently does not support this loss function...

## Steps to Reproduce
1. Generate synthetic regression data using make_regression.
2. Standardize X and normalize y.
3. Initialize plq_Ridge_Regressor with loss={'name': 'mse'}.
4. Call clf.fit(X, y).

## Minimal Code Example

```python
# Minimal reproducible example
import numpy as np
from rehline import ReHLine

import numpy as np
from sklearn.datasets import make_regression
from sklearn.preprocessing import StandardScaler
from rehline import plq_Ridge_Regressor

# 1. Simulate data
np.random.seed(42)
scaler_mse = StandardScaler()

n, d = 10000, 5
X, y = make_regression(n_samples=n, n_features=d, noise=1.0, random_state=42)
X = scaler_mse.fit_transform(X)
y = y / y.std()

# 2. Solve MSE Regression via plq_Ridge_Regressor
# This line causes the crash
clf = plq_Ridge_Regressor(loss={'name': 'mse'}, C=1.0)
clf.fit(X=X, y=y)
```

## Environment
- OS: Linux / 6.6.113+
- Python version: 3.12.12
- ReHLine version: 0.1.1
- NumPy version: 2.0.2
- SciPy version: 1.16.3
- scikit-learn version: 1.6.1
